TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -302,24 +302,24 @@ TYPO3:
       defaultEditPreviewMode: 'inPlace'
       editPreviewModes:
         # Live mode is only configured here for consistency. You shouldn't change it.
-        live:
+        'TYPO3.Neos:live':
```

Comment:
actually meant that prefixes all of them are breaking, not just live..

regarding only having it in the code would mean you cannot change the title